bazel/repositories.bzl

```diff
@@ -214,7 +215,7 @@ def _list_pl_deps(name):
                 best_url = url
         modules[best_url] = True
 
-    for repo_name, repo_config in GIT_REPOSITORY_LOCATIONS.items():
+    for _, repo_config in GIT_REPOSITORY_LOCATIONS.items():
```

Comment:
@kpattaswamy can you add a brief explanation here or in the PR description for why this was necessary? I think without context this could be confusing.